src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER = 100;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        int number = random.nextInt(NUMBER);
+        String color = colorSupplier.getRandomColor().name();
```

Comment:
you need to add `toLowerCase()` here, since `name()` returns you a String representation of the enum constant (`WHITE` instead of `white`)

```suggestion
        String color = colorSupplier.getRandomColor().name().toLowerCase();
```